packages/providers/src/provider.ts

```diff
@@ -243,7 +243,7 @@ export default class Provider {
    * @param options - Additional options for the provider
    * @hidden
    */
-  protected constructor(
```

Comment:
@LuizAsFight I'm curious - why did you remove the `protected` modifier from the constructor here?

Reply:
Yes, I'm assuming it was unintentional.

That should be kept.

Reply:
yep can remove it @Dhaiwat10